plugins/optimization-detective/class-od-preload-link-collection.php

```diff
@@ -136,6 +136,27 @@ static function ( array $carry, array $link ): array {
 		);
 	}
 
+	/**
+	 * Adds media features to the links.
+	 *
+	 * @phpstan-param Link $link
+	 * @param array $link Link array.
+	 * @phpstan-return Link
+	 * @return array Link array with media features added.
```

Comment:
```suggestion
	 * @param Link $link Link array.
	 * @return Link Array with media features added.
```

Reply:
Nevermind, I guess these are needed.

Reply:
Yes, I was looking for a better way but kept getting either phpstan or linter error without this.